### ISSUE TITLE
Adding charset UTF-8 to notifications

### ISSUE THREAD

**Issue by fgalan** [telefonicaid/fiware-orion#1340]
It seems a matter of using:
 
```  
Content-Type: application/json; charset=utf-8
```

instead of only

```
Content-Type: application/json
```

In notifications

**Comment by fgalan** [comment]
Implemented in PR https://github.com/telefonicaid/fiware-orion/pull/1345 (develop) and PR https://github.com/telefonicaid/fiware-orion/pull/1348.

However, the CNR entry was missed in the develop PR. Thus, let's keep this issue opened (as a reminder) to include that entry in a future PR.

**Comment by fgalan** [comment]
Also merged in release/0.22.0 in PR #1353

**Comment by fgalan** [comment]
CNR added in develop in PR https://github.com/telefonicaid/fiware-orion/pull/1357
